client/my-sites/email/email-providers-comparison/stacked/provider-cards/email-upsell-navigation.scss

```diff
@@ -3,13 +3,18 @@
 .email-upsell-navigation {
 	display: flex;
 	justify-content: space-between;
+	margin-bottom: 18px;
+
+	&.is-hiding-skip {
+		margin-bottom: 0;
+	}
 
 	.button {
 		@include onboarding-medium-text;
+
```

Comment:
Super NIT but... We don't need this space :)

Reply:
I double checked, but it looks like this is required now after the stylelint rules were recently updated.